### DISCUSSION TITLE
Input Dialog

### DISCUSSION THREAD

**Discussion by 1-Utkarsh-Pandey-1** [TomSchimansky/CustomTkinter#1822] [General]
Can I add iconbitmap in input dialog in custom tkinter If possible then please give solution

**Comment by Akascape** [comment]
@Utkarsh1233 
Have you tried this method?
```python
dialog = customtkinter.CTkInputDialog(text="Type in a number:", title="Test")
dialog.iconbitmap("icon.ico")
...
```

**Comment by ghost** [comment]
If you're on macOS, the `.iconbitmap` hardly ever works. It really only works on Windows.

**Reply by Akascape** [reply]
Yeah, that is why I like to use only iconphoto method. That one works in every platform.
